File: python/document.py
class Document:
    def __init__(self, document_id, document_type, document_path, application_id):
        self.document_id = document_id
        self.document_type = document_type
        self.document_path = document_path
        self.application_id = application_id

    def upload_document(self):
        print("Document uploaded.")

    def download_document(self, document_id):
        print(f"Document with ID {document_id} downloaded.")

    def validate_document(self):
        print("Document validated.")

File: GoaOnline/src/Document.java
public class Document {
    private int documentId;
    private String documentType;
    private String documentPath;
    private int applicationId;

    // Constructor
    public Document(int documentId, String documentType, String documentPath, int applicationId) {
        this.documentId = documentId;
        this.documentType = documentType;
        this.documentPath = documentPath;
        this.applicationId = applicationId;
    }

    // Methods
    public void uploadDocument() {
        System.out.println("Document uploaded.");
    }

    public void downloadDocument(int documentId) {
        System.out.println("Document with ID " + documentId + " downloaded.");
    }

    public void validateDocument() {
        System.out.println("Document validated.");
    }
}
